cli/azd/pkg/apphost/testdata/TestAspireBicepGeneration-main.bicep.snap

```diff
@@ -46,16 +46,16 @@ module ai 'ai/aspire.hosting.azure.bicep.appinsights.bicep' = {
     logAnalyticsWorkspaceId: resources.outputs.AZURE_LOG_ANALYTICS_WORKSPACE_ID
   }
 }
-module postgres2 'postgres2/aspire.hosting.azure.bicep.postgres.bicep' = {
-  name: 'postgres2'
+module postgres_2 'postgres-2/aspire.hosting.azure.bicep.postgres.bicep' = {
+  name: 'postgres-2'
   scope: rg
   params: {
     location: location
-    administratorLogin: administratorLogin
+    administratorLogin: administrator_login
     administratorLoginPassword: administratorLoginPassword
     databases: ['db2']
-    keyVaultName: resources.outputs.SERVICE_BINDING_POSTGRES2KV_NAME
-    serverName: 'postgres2'
+    keyVaultName: resources.outputs.SERVICE_BINDING_POSTGRES-2KV_NAME
```

Comment:
Is this working as expected here?

Reply:
I fixed it but forgot to update the snap :)
sometimes testing with snaps is not that cool, lol